native/src/utils/DatabaseConnector.ts

```diff
@@ -497,9 +526,16 @@ class DatabaseConnector {
           availableLanguages: jsonObject.availableLanguages,
           metaDescription: null, // not used in native
           excerpt: jsonObject.excerpt,
-          website: jsonObject.website,
-          phoneNumber: jsonObject.phoneNumber,
-          email: jsonObject.email,
+          contacts: jsonObject.contacts.map(
```

Comment:
:x: I think this will lead to errors/breaking behavior on first app start after these changes as `.contacts` is then undefined. So we can either
1. Increase the content version to force a new reload
2. Handle undefined values
3. Ignore this as the error is probably handled